atef/widgets/config/find_replace.py

```diff
@@ -56,7 +58,7 @@ def verify_file_and_notify(
     bool
         the verification success
     """
-    verified, msg = file.verify()
+    verified, msg = file.validate()
```

Comment:
Was this unknowingly broken for a few days?

Reply:
Yes 😢